homeassistant/components/climate/nest.py

```diff
@@ -97,6 +100,16 @@ def __init__(self, structure, device, temp_unit):
         self._min_temperature = None
         self._max_temperature = None
 
+    @property
+    def should_poll(self):
+        """Do not need poll thanks using Nest streaming API."""
+        return False
+
+    async def async_nest_update_event_handler(self, event):
```

Comment:
This should be a callback and not a coroutine. Decorate it with `@callback` imported from core.py.